tests/testthat/test_XcmsExperiment.R

```diff
@@ -1326,3 +1326,34 @@ test_that("storeResults,RDataParam works", {
     expect_s4_class(object, "XcmsExperiment")
     expect_equal(object, xmse)
 })
+
+test_that("storeResults,PlainTextParam,MsExperiment works", {
+    param <- PlainTextParam(path = "/path/test/")
```

Comment:
just to be on the safe side - also seeing that unit tests fail on the github actions: can you use here a path:

```
pth <- file.path(tempdir(), "test")
param <- PlainTextParam(path = pth)
```

Reply:
you will in fact get problems storing directly to the root directory `/` because of permissions.

Reply:
Ok good to know ! thanks.